cartographer/mapping/range_data_inserter_interface.h

```diff
@@ -37,6 +37,8 @@ class RangeDataInserterInterface {
   // Inserts 'range_data' into 'grid'.
   virtual void Insert(const sensor::RangeData& range_data,
                       GridInterface* grid) const = 0;
+
+  virtual ~RangeDataInserterInterface() {}
```

Comment:
this is the only important one, good catch! I personally like syntax:
`virtual ~RangeDataInserterInterface() = default` better though

Reply:
actually just realized you are re-declaring this, so should remove!